src/com/company/topic6/Main.java

```diff
@@ -0,0 +1,13 @@
+package com.company.topic6;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        ContulPersoanei contPersoanei = new ContulPersoanei();
+        contPersoanei.transferCatreOPersona("C:\\Users\\Admin\\Desktop\\Tema6\\SarcinaPractica6\\src\\com\\company\\aditional5\\Caini.java");
```

Comment:
better to use relative path, but it is ok
Example:
Relative path - src/com/tekwill/topic7
Absolute path - C:\\Users\\Admin\\Desktop\\Tema6\\SarcinaPractica6\\src\\com\\company\\aditional5